action.yml

```diff
@@ -142,70 +134,16 @@ inputs:
-    - ${{ inputs.cache-enable && '--cache-enable=' }}${{ inputs.cache-enable }}
-    - ${{ inputs.cert && '--cert=' }}${{ inputs.cert }}
-    - ${{ inputs.config && '--config=' }}${{ inputs.config }}
-    - ${{ inputs.context-dir && '--context-dir=' }}${{ inputs.context-dir }}
-    - --context-type=github
-    - ${{ inputs.crl && '--crl=' }}${{ inputs.crl }}
-    - ${{ inputs.crl-full-chain && '--crl-full-chain=' }}${{ inputs.crl-full-chain }}
-    - ${{ inputs.deliverable && '--deliverable=' }}${{ inputs.deliverable }}
-    - ${{ inputs.disable-crl && '--disable-crl=' }}${{ inputs.disable-crl }}
-    - ${{ inputs.env && '--env=' }}${{ inputs.env }}
-    - ${{ inputs.filter-regex && '--filter-regex=' }}${{ inputs.filter-regex }}
-    - ${{ inputs.filter-scope && '--filter-scope=' }}${{ inputs.filter-scope }}
-    - ${{ inputs.git-branch && '--git-branch=' }}${{ inputs.git-branch }}
-    - ${{ inputs.git-commit && '--git-commit=' }}${{ inputs.git-commit }}
-    - ${{ inputs.git-tag && '--git-tag=' }}${{ inputs.git-tag }}
-    - ${{ inputs.key && '--key=' }}${{ inputs.key }}
-    - ${{ inputs.label && '--label=' }}${{ inputs.label }}
-    - ${{ inputs.level && '--level=' }}${{ inputs.level }}
-    - ${{ inputs.log-context && '--log-context=' }}${{ inputs.log-context }}
-    - ${{ inputs.log-file && '--log-file=' }}${{ inputs.log-file }}
-    - ${{ inputs.oci && '--oci=' }}${{ inputs.oci }}
-    - ${{ inputs.oci-repo && '--oci-repo=' }}${{ inputs.oci-repo }}
-    - ${{ inputs.output-directory && '--output-directory=' }}${{ inputs.output-directory }}
-    - ${{ inputs.output-file && '--output-file=' }}${{ inputs.output-file }}
-    - ${{ inputs.pipeline-name && '--pipeline-name=' }}${{ inputs.pipeline-name }}
-    - ${{ inputs.policy-args && '--policy-args=' }}${{ inputs.policy-args }}
-    - ${{ inputs.predicate-type && '--predicate-type=' }}${{ inputs.predicate-type }}
-    - ${{ inputs.product-key && '--product-key=' }}${{ inputs.product-key }}
-    - ${{ inputs.product-version && '--product-version=' }}${{ inputs.product-version }}
-    - ${{ inputs.scribe-auth-audience && '--scribe.auth.audience=' }}${{ inputs.scribe-auth-audience }}
-    - ${{ inputs.scribe-client-id && '--scribe.client-id=' }}${{ inputs.scribe-client-id }}
-    - ${{ inputs.scribe-client-secret && '--scribe.client-secret=' }}${{ inputs.scribe-client-secret }}
-    - ${{ inputs.scribe-enable && '--scribe.enable=' }}${{ inputs.scribe-enable }}
-    - ${{ inputs.scribe-login-url && '--scribe.login-url=' }}${{ inputs.scribe-login-url }}
-    - ${{ inputs.scribe-url && '--scribe.url=' }}${{ inputs.scribe-url }}
-    - ${{ inputs.structured && '--structured=' }}${{ inputs.structured }}
-    - ${{ inputs.timeout && '--timeout=' }}${{ inputs.timeout }}
-    - ${{ inputs.verbose && '--verbose=' }}${{ inputs.verbose }}
+  using: composite
+  steps:
+    - shell: bash
+      run: curl -sSfL https://get.scribesecurity.com/install.sh  | sh -s -- -t valint
```

Comment:
Can we add a condition on installing valint only when binary not found in default path?

Reply:
maybe we should update the script instead wdyt?

Reply:
I prefer to effecting all curl scripts with this logic